### DISCUSSION TITLE
gkd怎么保证后台杀死app权限不掉的呢

### DISCUSSION THREAD

**Discussion by WilsonScott** [gkd-kit/gkd#826] [Q&A] [answered]
我的app从后台杀死app后无障碍立即就掉了，gkd和autojs 都不会掉，我这边也设置了前台服务，也有通知权限和自启动权限，表现行为却不一样。。。

**Comment by lisonge** [comment, marked as answer]
在**有无障碍服务运行**的情况下

用户划掉任务界面的卡片并不会停止进程

**Reply by WilsonScott** [reply]
感谢大佬回复，我这边的表现是点击menu菜单划掉后，再去设置界面看无障碍确实已经掉了，app没有任务crash日志，确实很奇怪。
